zivid_samples/src/sample_capture.cpp

```diff
@@ -1,113 +1,85 @@
-// Copyright 2024 Zivid AS
```

Comment:
it was one of the ament lint checks that required all files to have a copyright statement. Since you enabled more linting of samples, it is weird that this copyright is not needed here. Are we doing same linting on zivid_camera and zivid_samples? I dont mind not having the copyright here, but we should verify that we are using the same linting across the packages.

Reply:
Yes, I removed the copyright linter and also cpplint which had a separate license check. I think it should be okay to remove the latter one too, considering this: https://github.com/google/styleguide/pull/837, and I added clang_format and clang_tidy, I think those two tools should cover the important things from here.

I didn't remove the copyright linter from zivid_camera, since I didn't see the need to remove the copyright notice here. But if we want to be consistent I can remove it from there too.